### PR TITLE
LRDOCS-7788 search 7.3 | upgrading elasticsearch - sme edits vol.3.

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-7788

Hey Russ,

I think I was able to clean this up. See my changes:https://github.com/lipusz/liferay-learn/compare/f75688b0f1b84e127ec81570cd913cc73dcf3952...pr-18-LRDOCS-7788

I have removed a couple of sections that we do not need in my opinion. For example, we have never supported ES 5.x.

Hope this will be good now.

Cheers,
Tibor

### CODE REVIEW

**Comment by rbohl** [comment]
Merged locally.

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 Upgrading Liferay requires some additional steps to upgrade the search experience:
 
-- Backing up your indexes
-- Installing Elasticsearch 7 (and the plugins Liferay requires)
+- Backing up your Elasticsearch indexes
+- Installing Elasticsearch 7.9+ (and the plugins Liferay requires)
 - Connecting Liferay to Elasticsearch (may require a new connector)
 - Re-indexing the Workflow Metrics, Company, and System indexes from their dedicated user interfaces
-- Restoring the search tuning indexes from the backed up version
+- Restoring the Search Tuning indexes from a snapshot (when upgrading from Liferay DXP 7.2 FP2+/SP1+)
```
Comment (rbohl) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:9]: @lipusz if you did a rolling upgrade or a full cluster restart upgrade, you wouldn't need to restore from snapshot would you? Just want to make sure this is the best approach that we are recommending.

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:9]: Yes.

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:9]: Sorry, did you mean

"yes, you would still need tor estore from snapshot"
OR
"yes, that's right, you wouldn't need to restore from snapshot"
:stuck_out_tongue_winking_eye:

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:9]: Russ: I believe I have already explained that enough times.

If you upgrade your existing Elasticsearch installation it is expected to preserve whatever indexes you had before the upgrade. That's what upgrading means implicitly.

When you set up a new ES installation, it's obvious that those index data that are not re-indexable from the portal (Search Tuning) needed to be resorted manually.

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:9]: :sweat_smile:  Thanks for continuing to explain. The reason I keep bringing it up is that in this section we mix recommending the rolling (or full cluster restart if necessary) upgrades with this information on restoring from a snapshot, and I can't figure out why.

You're the SME, so if you find this clear, I'll simply push it forward.

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-learn/pull/299

**Pull request closed without merging**